client/src/components/Resource/ResourceForecast.js

```diff
@@ -15,96 +18,116 @@ class Forecast extends Component {
                     </tr>
                     </thead>
                     <tbody>
-                    <tr>
-                        <td>
-                            <a href="/project/:projectCode">Placeholder</a>
-                        </td>
-                        <td>
-                            <a href="/project/:projectCode/:changeItem">
-                                Placeholder
-                            </a>
-                        </td>
-                        <td>
-                            <a href="/project/:projectCode/:changeItem/:requiredResource">
-                                Placeholder
-                            </a>
-                        </td>
-                        <td>Placeholder</td>
-                    </tr>
+                    {this.props.projects.forEach(project => {
+                        project.changeItems.forEach(changeItem => {
+                            totalDays += changeItem.resourcesRequired.impact.days;
+                            return (
```

Comment:
Again - this 'return' section should be separated out into it's own component and rendered that way